plaidml/bridge/openvino/ops/roi_pooling.cpp

```diff
@@ -80,46 +81,13 @@ edsl::Tensor bilinear_pooling(edsl::Tensor I, const std::vector<float>& coord, i
   float roi_w_scale = roi_width / (pooled_w - 1);
 
   // get center point of every ROI bin.
-  auto in_y = edsl::cast(edsl::index({edsl::TensorDim(pooled_h)}, 0), DType::FLOAT32) * roi_h_scale +
+  auto in_h = edsl::cast(edsl::index({edsl::TensorDim(pooled_h)}, 0), DType::FLOAT32) * roi_h_scale +
               roi_h_start * (height - 1);
-  auto in_x =
+  auto in_w =
       edsl::cast(edsl::index({edsl::TensorDim(pooled_w)}, 0), DType::FLOAT32) * roi_w_scale + roi_w_start * (width - 1);
 
-  // get it's interpolation point.
-  auto top_y_index = edsl::floor(in_y);
-  auto bottom_y_index = edsl::ceil(in_y);
-  auto left_x_index = edsl::floor(in_x);
-  auto right_x_index = edsl::ceil(in_x);
-
-  // check it border.
-  auto h_border = edsl::cast(edsl::Tensor(height - 1), DType::FLOAT32);
-  auto w_border = edsl::cast(edsl::Tensor(width - 1), DType::FLOAT32);
-  bottom_y_index = edsl::select(bottom_y_index < h_border, bottom_y_index, h_border);
-  right_x_index = edsl::select(right_x_index < w_border, right_x_index, w_border);
-
-  // gather the interpolation point from feature map.
-  auto get_pieces = [](edsl::Tensor temp, edsl::Tensor y, edsl::Tensor x) {
-    auto gather_x = edsl::gather(temp, x).axis(3);
-    auto gather_y = edsl::gather(gather_x, y).axis(2);
-    return gather_y;
-  };
-  edsl::Tensor top_left = get_pieces(I, top_y_index, left_x_index);
-  edsl::Tensor top_right = get_pieces(I, top_y_index, right_x_index);
-  edsl::Tensor bottom_left = get_pieces(I, bottom_y_index, left_x_index);
-  edsl::Tensor bottom_right = get_pieces(I, bottom_y_index, right_x_index);
-
-  std::vector<edsl::TensorDim> dims(I.rank());
-  I.bind_dims(dims);
-  dims[2] = edsl::TensorDim(pooled_h);
-  dims[3] = edsl::TensorDim(pooled_w);
-  auto top = top_left + (top_right - top_left) * (in_x - left_x_index);
-  auto bottom = bottom_left + (bottom_right - bottom_left) * (in_x - left_x_index);
-
-  // final formula : output = top + (bottom - top) * (in_y - top_y_index)
-  // we have to reshape mul operands.
-  auto temp_shape = edsl::reshape(bottom - top, {dims[0], dims[1], dims[3], dims[2]});
-  auto output = top + edsl::reshape(temp_shape * (in_y - top_y_index), dims);
-  return output;
+  auto I_gathered_h = edsl::gather(I, in_h).axis(2);
```

Comment:
I used to use our gather op like this on commit fae3121115e4171a379b01bc0e5ac2e94a2947f8. then it can't pass all the test cases, so I change it to complex one. Now I understand, this may caused by openvino `setup`, after fix openvino side it's alright.  
thanks a lots to fix this 😂😂😂